app/controllers/pieces_controller.rb

```diff
@@ -1,3 +1,27 @@
 class PiecesController < ApplicationController
 
```

Comment:
standard show/update/params controller actions with my tickets for calling valid_move? and move_to? within the update action. I will be writing RSpec tests for these actions as soon as we can actually test this functionality. But for now, they don't seem to break of our current functionality.